apps/apps.humanatlas.io/src/app/pages/us1/cell-population-predictions/cell-population-predictions.component.ts

```diff
@@ -0,0 +1,159 @@
+import { CommonModule } from '@angular/common';
+import { ChangeDetectionStrategy, Component, effect, input, viewChild, HostListener, inject } from '@angular/core';
+import { MatIconModule } from '@angular/material/icon';
+import { MatSort, MatSortModule } from '@angular/material/sort';
+import { MatTableDataSource, MatTableModule } from '@angular/material/table';
+import { DeleteFileButtonComponent } from '@hra-ui/design-system/buttons/delete-file-button';
+import { WorkflowCardModule } from '@hra-ui/design-system/workflow-card';
+import { MatMenuModule } from '@angular/material/menu';
+import { TooltipCardComponent, TooltipContent } from '@hra-ui/design-system/tooltip-card';
+import { OverlayModule } from '@angular/cdk/overlay';
+import { TOOLTIP_POSITION_RIGHT_SIDE } from '@hra-ui/cde-visualization';
+import { Router } from '@angular/router';
+
+/**
+ * Prediction Result
+ */
+export interface Prediction {
```

Comment:
Move this interface into the predictions service file, otherwise you'll get circular import errors